riscv.c

```diff
@@ -803,23 +1304,39 @@ void rv_step(struct riscv_t *rv, int32_t cycles)
         OP(madd),   OP(msub),     OP(nmsub), OP(nmadd),    OP(fp),     OP(unimp), OP(unimp), OP(unimp), // 10
         OP(branch), OP(jalr),     OP(unimp), OP(jal),      OP(system), OP(unimp), OP(unimp), OP(unimp), // 11
     };
-// clang-format on
+    TABLE_TYPEC jump_tablec[] = {
+        //  00              01              10          11
```

Comment:
Align the comment to ensure "00", "01", "10", and "11" starting with the corresponding `OP` positions.